lib/kitchen/driver/vagrant.rb

```diff
@@ -389,6 +402,12 @@ def update_state(state)
         state[:rdp_port] = hash["RDPPort"] if hash["RDPPort"]
       end
 
+      # @return [String] full absolute path to the kitchen cache directory
+      # @api private
+      def local_kitchen_cache
+        @local_kitchen_cache ||= File.expand_path('~/.kitchen/cache')
```

Comment:
@mwrock Apologize Matt but I couldn't understand your concern about the cache directory in the scope of testing multiple cookbooks, the cache is uniq per user, that is why we are storing it inside the home directory, that means that it will be shared across suites, instances and even projects (different repositories). 

Another important thing to mention is that this cache is for chef packages and not cookbooks, maybe that cache that you mention is the actual berkshelf cache which is different.

Let me know if I am helping with this comment or confusing you more. 😄